users/tests/test_users_views.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from django.urls import reverse
+from django.test import Client
+from pytest_django.asserts import assertTemplateUsed
+from django.contrib.auth.models import User
+
+
+class TestViews:
+    def setup(self):
+        self.client = Client()
+
+    @pytest.fixture
+    def user(self):
+        return User.objects.create_user('Test-user', 'Test-user@mail.com', 'Password')
+
+    def test_register_POST(self):
```

Comment:
Please use this fixture for views test,
 @fixture https://pytest-django.readthedocs.io/en/latest/helpers.html#rf-requestfactory 
 with this django test tool:
 https://docs.djangoproject.com/en/3.2/topics/testing/advanced/#django.test.RequestFactory